job-dsls/jobs/kie/main/kie_jenkinsScripts_PR.groovy

```diff
@@ -2,8 +2,7 @@ import org.kie.jenkins.jobdsl.Constants
 
 // definition of parameters
 
-def javaToolEnv="KIE_JDK11_HOME"
-def mvnToolEnv="KIE_MAVEN_3_8_1_HOME"
+def mvnToolEnv=Constants.MAVEN_TOOL
 def repoBranch=Constants.BRANCH
 def organization=Constants.GITHUB_ORG_UNIT
 def javadk=Constants.JDK_VERSION
```

Comment:
and then (based on the Constants file comment)
```suggestion
def javaToolEnv=Constants.JDK_TOOL
```

Reply:
DONE.
All JAVA_VERSION were changed to JAVA_TOOL